server.go

```diff
@@ -41,7 +41,13 @@ type errorResponse struct {
 // errorMapper maps an error from a handler into an HTTP server error.
 // TODO map other errors
 var errorMapper httprequest.ErrorMapper = func(err error) (int, interface{}) {
-	return http.StatusInternalServerError, &errorResponse{
+	status := http.StatusInternalServerError
+	cause := errgo.Cause(err)
+	switch cause {
+	case params.ErrUnparsable:
+		status = 422 // "Unprocessable Entity" - http doesn't have a const for this error.
+	}
+	return status, &errorResponse{
```

Comment:
Probably for a subsequent PR but you probably want to define errorResponse publicly in params. That way you can check for it specifically in this mapper, and that gives API endpoints the freedom to return custom errors directly (you could add an "Info" field to hold all the individual verification errors, for example).